src/organizations/OrganizationEntity.js

```diff
@@ -159,6 +159,9 @@ export class OrganizationEntity extends Entity {
 	copyCoursePageUrl() {
 		return this._getHref(Rels.copyCoursePage);
 	}
+	outcomesSettingsUrl() {
+		return this._getHref(Rels.Outcomes.settings);
+	}
```

Comment:
missing a newline! Scandalous

Reply:
uhm _ackstually_ 

I did do this on purpose, because these `*Url()` functions were already clumped together! I didn't start the fire, I'm just adding fuel to it and not asking questions about why it's there in the first place.